src/transports/abstract/subscription.ts

```diff
@@ -68,7 +78,16 @@ export abstract class SubscriptionTransport<const T extends TransportGenerics>
       })
       .set(entries.length)
 
-    await this.backgroundHandler(context, entries)
+    try {
+      await this.backgroundHandler(context, entries)
+      this.retryCount = 0
```

Comment:
This is tracking state in the transport itself and there is a non-zero chance that a call to the bg handler takes more than the timeout and the framework will retry before that promise resolves; not critical here in practice but worth mentioning

Reply:
I see the point, although I agree I don't think this is an issue here. The retry only matters if the handler will error out, but I don't think we want to change `retryCount` and `retryTime` until we _know_ that a `backgroundHandler` call has failed. The biggest risk is if network requests take a while to respond in the `backgroundHandler` and _then_ it fails, but at that point it will increment the `retryCount` and start limiting additional requests.